components/Ripple/README.md

```diff
@@ -360,3 +362,52 @@ MDCStatefulRippleView *statefulRippleView = [[MDCStatefulRippleView alloc] init]
 ```
 <!--</div>-->
 
+
+## Migrations
+
+<!-- Extracted from docs/migration-usesSuperviewShadowLayerAsMask.md -->
+
+### usesSuperviewShadowLayerAsMask migration
+
+tl;dr: If you are adding ripples to views with custom `layer.shadowPath` values, please disable
+`usesSuperviewShadowLayerAsMask` and assign an explicit layer mask to the ripple view if needed
+instead. usesSuperviewShadowLayerAsMask will eventually be disabled by default and then deleted.
+
+MDCRippleView currently implements a convenience behavior that will inherit its parent view's
+`layer.shadowPath` as the mask of the ripple view itself. This works for the general case where the
+ripple view's frame equals the bounds of its super view, but behaves unexpectedly for any other
+frame of the ripple view.
+
+Due to the brittleness of this behavior, a new migration property, `usesSuperviewShadowLayerAsMask`,
+has been added that will allow you to disable this behavior in favor of a more explicit
+determination of the ripple's layer mask.
+
+Example usage:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
```

Comment:
I *think* you need more empty lines between these elements. There's no way to know, so it's a bit of [Cargo Cult Programming](https://en.wikipedia.org/wiki/Cargo_cult_programming), but I think the div + headers get messed-up when they're rendered on material.io.

See https://material.io/develop/ios/components/tabs/#theming-extensions which is generated from https://github.com/material-components/material-components-ios/tree/develop/components/Tabs#theming-extensions.

Reply:
Done.